app/services/podcasts/episode_rss_item.rb

```diff
@@ -25,8 +25,8 @@ def self.from_item(item)
     end
 
     def to_h
-      ATTRIBUTES.each_with_object({}) do |key, hash|
-        hash[key] = instance_variable_get("@#{key}")
+      ATTRIBUTES.index_with do |key|
```

Comment:
I never knew about `index_with` neat!

Reply:
@mstruve it was added in Rails 6, I didn't either :D